setup.py

```diff
@@ -66,12 +65,11 @@ def run_tests(self):
 # Extract name and e-mail ("Firstname Lastname <mail@example.org>")
 AUTHOR, EMAIL = re.match(r'(.*) <(.*)>', AUTHOR_EMAIL).groups()
 
-# For some versions of python (2.6) we need the importlib
-# as it is not in the standard install
-try:
-    import importlib
-except ImportError:
-    REQUIREMENTS.append('importlib')
+REQUIREMENTS = ['requests', 'dicttoxml']
```

Comment:
Is there a reason for not reading them from the file. There is always the risk of getting out of sync!